HW0/P4/P4.py

```diff
@@ -40,37 +44,108 @@
     # Read the observation array and plot it (Part 2)
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    s_m = np.loadtxt('P4_measurements.txt', delimiter=',')
+    x_coords = 1.0 / rx * s_m[:,0]
+    y_coords = 1.0 / ry * s_m[:,1]
+    z_coords = 1.0 / rz * s_m[:,2]
+    ax.plot(x_coords, y_coords, z_coords,
+             '.g', label='Observed trajectory')
 
     #####################
     # Part 3:
     # Use the initial conditions and propagation matrix for prediction
     #####################
 
-    # A = ?
-    # a = ?
-    # s = ?
+    A = np.identity(6)
+    A[0:3,3:6] = np.identity(3) * dt
+    A[3:6,3:6] = np.identity(3) - c * dt * np.identity(3)
```

Comment:
I like how you used `np.identity` rather than hardcoding, makes the code cleaner!